src/styles/sidebar/components/toast-messages.scss

```diff
@@ -0,0 +1,83 @@
+@use "../../mixins/panel";
+@use "../../variables" as var;
+
+.toast-messages {
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  left: 0;
+  padding: 0 8px;
+}
+
+.toast-message-container {
+  position: relative;
+  width: 100%;
+  animation: slidein 0.3s forwards ease-in-out, fadeout 0.5s 4.5s;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.toast-message {
+  @include panel.panel;
+  display: flex;
+  position: relative;
+  width: 100%;
+  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.15);
```

Comment:
I think its time for a much needed app-wide variable here. Maybe a few standard iterations?

Reply:
Yes, this is on my mental list. Given the size of this diff I didn't want to try to optimize out the CSS components and mixins just yet.